.github/workflows/ScriptTests.yml

```diff
@@ -22,28 +22,30 @@ jobs:
         steps:
             - name: "Checking changed files"
               id: files
-              uses: jitterbit/get-changed-files@v1
+              uses: softprops/diffset@v1
+              with:
+                global_files: |
+                  !packages/**/*
+              env:
+                GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
             - name: "Checking-out code"
               uses: actions/checkout@v2
               with:
                   submodules: false
             - name: "Fetching master to compare"
```

Comment:
Do we need to fetch here? Is `softprops/diffset@v1` not doing that already to see file changes?

Reply:
The action doesnt fetch the branch, it acquires the changed files from the PR.
Lerna requires master to be fetchedt only when is a PR that didnt affect the global files to do partial building